javaagent-extension-api/src/test/java/io/opentelemetry/javaagent/extension/internal/DeclarativeConfigPropertiesBridgeTest.java

```diff
@@ -24,51 +22,31 @@
 
 class DeclarativeConfigPropertiesBridgeTest {
 
-  private static final String YAML =
-      "file_format: 0.4\n"
-          + "instrumentation/development:\n"
-          + "  java:\n"
-          + "    common:\n"
-          + "      default-enabled: true\n"
-          + "    runtime-telemetry:\n"
-          + "      enabled: false\n"
-          + "    example-instrumentation:\n"
-          + "      string_key: value\n"
-          + "      bool_key: true\n"
-          + "      int_key: 1\n"
-          + "      double_key: 1.1\n"
-          + "      list_key:\n"
-          + "        - value1\n"
-          + "        - value2\n"
-          + "        - true\n"
-          + "      map_key:\n"
-          + "        string_key1: value1\n"
-          + "        string_key2: value2\n"
-          + "        bool_key: true\n"
-          + "    acme:\n"
-          + "      full_name:\n"
-          + "        preserved: true";
-
   private ConfigProperties bridge;
   private ConfigProperties emptyBridge;
 
   @BeforeEach
   void setup() {
-    OpenTelemetryConfigurationModel model =
-        DeclarativeConfiguration.parse(
-            new ByteArrayInputStream(YAML.getBytes(StandardCharsets.UTF_8)));
-    SdkConfigProvider configProvider = SdkConfigProvider.create(model);
-    bridge =
-        new DeclarativeConfigPropertiesBridge(
-            Objects.requireNonNull(configProvider.getInstrumentationConfig()));
+    bridge = create(new DeclarativeConfigPropertiesBridgeBuilder());
 
     OpenTelemetryConfigurationModel emptyModel =
         new OpenTelemetryConfigurationModel()
             .withAdditionalProperty("instrumentation/development", new InstrumentationModel());
     SdkConfigProvider emptyConfigProvider = SdkConfigProvider.create(emptyModel);
     emptyBridge =
-        new DeclarativeConfigPropertiesBridge(
-            Objects.requireNonNull(emptyConfigProvider.getInstrumentationConfig()));
+        new DeclarativeConfigPropertiesBridgeBuilder()
+            .buildFromInstrumentationConfig(
+                Objects.requireNonNull(emptyConfigProvider.getInstrumentationConfig()));
```

Comment:
oh, can getInstrumentationConfig() return null? would it make sense for it to return empty instead to avoid null checks?

Reply:
I agree that empty would be nicer to use - but that's a discussion for the SDK

Reply:
`InstrumentationConfig` mitigates this problem it the java agent and spring starter